content/millennium/r4/financial/financial-transaction.md

```diff
@@ -0,0 +1,209 @@
+
+## Terminology Bindings
+
+<%= terminology_table(:financial_transaction, :r4) %>
+
+## Extensions
+
+* [Financial Transaction Account Number]
+* [Financial Transaction Alias]
+* [Financial Transaction Allocation]
+* [Financial Transaction Amount]
+* [Financial Transaction Card Brand]
+* [Financial Transaction Date]
+* [Financial Transaction Location]
+* [Financial Transaction Method]
+* [Financial Transaction Tendered Amount]
+* [Financial Transaction Type]
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
+
+ ID                                      | Value\[x] Type      | Description
+-----------------------------------------|---------------------|--------------------------------------------------------------------------
+ `financial-transaction-account-number`  | [`string`]          | A value associated to the specific payment method usually represented as the last four digits of a credit card, the check number, the EFT number or Lockbox number.
+ `financial-transaction-alias`           | [`string`]          | Client defined value to represent the combination of the type, subtype, and reason describing the financial transaction.
+ `financial-transaction-allocation`      | None (contains nested extensions)      | Defines how the payment or adjustment is to be allocated across other resources.
+ `financial-transaction-amount`          | [`Money`]           | The total amount of the financial transaction.
+ `financial-transaction-card-brand`      | [`string`]          | Identifies the brand of credit card when credit card is used as a payment method.
+ `financial-transaction-date`            | [`date`]            | Represents the expiration date if method is card, check date if method is check, EFT date if method is EFT or Lockbox date if method is lockbox.
+ `financial-transaction-location`        | [`string`]          | Client configured value representing the location or workflow that the payment was received in.
+ `financial-transaction-method`          | [`string`]          | Describes the method of payment for the financial transaction.
+ `financial-transaction-tendered-amount` | [`Money`]           | The amount of cash originally tendered for payment. This value should be greater than or equal to the amount of the cash payment.
+ `financial-transaction-type`            | [`CodeableConcept`] | The classification of the transaction.
+
+## Create
+
+Create a FinancialTransaction.
+
+    POST /Basic    
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported.
+* When integrating your application with a client's production environment, the client will have to provide the appropriate financial-transaction-alias values to send with the financial transaction.
+* The financial-transaction-allocation.target field must reference an Encounter associated with an Account.
```

Comment:
let's remove this line and move it into the body fields. We have a line below that explains it better, and we can put this in the notes on the body fields.

Reply:
The body field currently has type, method and amount. Not sure under which of these you would like to add the note to.

Reply:
This should be moved to the allocation.target field. 

For clarity, this is the line that should be moved. (The financial-transaction-allocation.target field must reference an Encounter associated with an Account.)
The statement on line 60 seems to describe it more eloquently.

Reply:
Removed that line here
https://github.com/cerner/fhir.cerner.com/pull/297/commits/88846cda493e3b206f9514953069ea942c524867#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cL58

and added it to the target field
cf0fb07#diff-26870e1b94fc9955e130f3f940dbe2e7R273

Reply:
+1